base/MAPL_IO.F90

```diff
@@ -7370,8 +7372,17 @@ subroutine MAPL_BundleReadNCPar(Bundle, arrdes, filename, rc)
        _VERIFY(status)
        grid_file_match=compare_grid_file(metadata,grid,rc=status)
        _VERIFY(status)
+       flip = check_flip(metadata,rc=status)
```

Comment:
No need to change now,  but please do use these types of opportunities to introduce the `_RC` macro.